src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -292,6 +294,19 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = (props) => {
 
   const defaultVenue = createJazzbar({});
 
+  const img = new Image();
```

Comment:
I believe that currently, this image object will be created on every re-render of the functional component (which would then re-trigger the `useEffect`, etc). Is that the desired/intended state?

If it was just created 'outside' of the `useEffect` for 'variable scope' reasons, you could probably use `useRef` or similar to 'store' it in a persistent way.

- https://reactjs.org/docs/hooks-reference.html#useref

Reply:
I just moved it inside the useEffect which has ```[venue.columns, venue.mapBackgroundImageUrl]``` as dependency array. Now new image is going to get created only when necessary.